dotnet/src/Connectors/Connectors.Memory.AzureCosmosDBNoSQL/AzureCosmosDBNoSQLConstants.cs

```diff
@@ -6,4 +6,7 @@ internal static class AzureCosmosDBNoSQLConstants
 {
     /// <summary>Reserved key property name in Azure CosmosDB NoSQL.</summary>
     internal const string ReservedKeyPropertyName = "id";
+
+    /// <summary>Variable name for table in Azure CosmosDB NoSQL queries.</summary>
+    internal const string TableQueryVariableName = "x";
```

Comment:
Curious about the default value of `"x"`. Can you add remarks to the comment explaining why this is the proper default?

Reply:
This can be any string, which identifies a table in SQL query, for example `SELECT x.Name FROM x`. Updated XML doc.